### PR TITLE
refactoring/s5erasemipmap

### PR DESCRIPTION
**Pull request by davidberard2**
Erased unused mipmap and refactored Manifest

### CODE REVIEW

**Comment by davidberard2** [comment]
Hi Kayne and Alex, there is the answer to your question. The icon should work as usual.

**Review by Alex-Hayeur — APPROVED** [review]
The Icon is still functional for me, so reducing the amount of unnecessary files is great. Thanks for the fast work.

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
     <application
         android:allowBackup="true"
-        android:icon="@drawable/xhdpi"
+        android:icon="@mipmap/ic_launcher_round"
```
Comment (kkhh001) [inline app/src/main/AndroidManifest.xml:16]: Nice tweak to make the icon adaptive !

**Review by kkhh001 — APPROVED** [review]
The app runs as it should after removing these files. Good job on the refactoring!

**Pull request merged**